### ISSUE TITLE
Exposure compensation adjustments has no effect

### ISSUE THREAD

**Issue by dpashkevich** [xplodwild/android_packages_apps_Focal#5]
Using Sony Xperia S running CyanogenMod 10 (from FreeXperia, FXP2360. Happy to provide any additional debug information if you need it.

**Comment by dpashkevich** [comment]
Trying out version 1.0-dfa5ee8: now the exposure adjustment affects the preview (both liveview and fast preview of the taken shot) but still has no effect on the actual image taken.

**Comment by xplodwild** [comment]
This sounds like a device issue, I'll test on Xperia T

**Comment by dpashkevich** [comment]
I don't think so. I just tested exposure compensation in [CameraJB+](https://play.google.com/store/apps/details?id=com.moblynx.camerajbplus) (ported stock Android 4.3 camera with some extra features) and it works there.
